apps/frontend/src/libs/hooks/use-pagination/libs/enums/query-parameter.enum.ts

```diff
@@ -0,0 +1,6 @@
+const QueryParameter = {
```

Comment:
let's call it `QueryParameterName` and move to `apps/frontend/src/libs/enums/query-parameter-name.enum.ts`
```suggestion
const QueryParameterName = {
```